bundle/pom.xml

```diff
@@ -63,9 +63,12 @@
                             org.apache.poi.ss.usermodel;version="[2.0,4)",  <!-- using a wider version range for forward compatibility -->
                             org.apache.poi.ss.util;version="[2.0,4)",
                             org.apache.poi.xssf.usermodel;version="[2.0,4)",
+                            org.bouncycastle.jce;resolution:=optional,  <!-- jjwt -->
```

Comment:
I suspect that all three of these lines should not be optional imports and should just be excluded from the import list.

android.util is self-evident -- there's no use case where this would be installed in AEM

According to the README on https://github.com/jwtk/jjwt, BouncyCastle is only used for RSASSA-PSS signatures. This code only uses RSASSA-PKCS and so BouncyCastle is both not required and doesn't add any value.

Reply:
without these lines I not able to get the bundle to the active-state

Reply:
right. I'm suggesting to exclude the imports entirely with something like

    !org.bouncycastle.*,
    !android.*

See https://bnd.bndtools.org/heads/import_package.html

Reply:
right, got it. I applied it like this